demisto_sdk/commands/generate_test_playbook/README.md

```diff
@@ -5,14 +5,17 @@ Generate Test Playbook from integration/script yml
 * **-i, --input**
    Specify integration/script yml path (must be a valid yml file)
 * **-o, --output**
-   Specify output directory (Default: current directory)
+   Specify output directory or path. If not specified, and the input is located at `.../Packs/<pack_name>/Integrations`, the output is saved under `.../Packs/<pack_name>/TestPlaybooks`. If no folder in the input hierarchy is named Packs, the output is saved in the current directory.
```

Comment:
```suggestion
   Specify output directory or path. If not specified, and the input is located at `...content/Packs/<pack_name>/Integrations`, the output is saved under `...content/Packs/<pack_name>/TestPlaybooks`. If no folder in the input hierarchy is named Packs, the output is saved in the current directory.
```

Reply:
What do you think about this command being called by other repositories than content? e.g content-dist-utils, think it is a little incorrect to put content/Packs...

Reply:
_by_ other repos, or on input _located_ under other repos, with a path that doesn't have `content/Packs` in it?